config/import-control-test.xml

```diff
@@ -20,13 +20,16 @@
   <!-- Disallow @BeforeAll, see https://github.com/checkstyle/checkstyle/issues/12100 -->
   <disallow class="org.junit.jupiter.api.BeforeAll"/>
   <!-- Disallow Junit assert in favor of Truth asserts in few exceptions -->
-  <allow class="org.junit.jupiter.api.Assertions.assertThrows"/>
   <allow class="org.junit.jupiter.api.Assertions.assertDoesNotThrow"/>
   <disallow pkg="org.junit.jupiter.api.Assertions"/>
 
   <!-- Reflection shouldn't be used in tests. -->
   <disallow pkg="java\.lang\.reflect\.*" regex="true" />
 
+    <!-- https://github.com/checkstyle/checkstyle/issues/11343 -->
+  <disallow pkg="org.junit.jupiter.api.Assertions.assertThrows"/>
```

Comment:
Align xml.
Update comment to `see reason at ...`

Reply:
Done